internal/resource/config/oauth/client/oauth_client_resource.go

```diff
@@ -1280,6 +1279,19 @@ func (r *oauthClientResource) ModifyPlan(ctx context.Context, req resource.Modif
 	}
 
 	planModified := false
+
+	if !pfVersionAtLeast120 &&
+		!pfVersionAtLeast121 &&
+		!pfVersionAtLeast122 {
+		if internaltypes.IsDefined(plan.PersistentGrantExpirationTime) {
+			plan.PersistentGrantExpirationTime = types.Int64PointerValue(plan.PersistentGrantExpirationTime.ValueInt64Pointer())
```

Comment:
I don't think this is doing anything since we know at this point that the value is not null or unknown.